Dockerfile

```
@@ -11,15 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM quay.io/konveyor/builder as builder
+FROM quay.io/konveyor/builder:ubi9-latest AS builder
```

@mateusoliveira43 do you also want this tag reverted?
then they would be both centos/rhel8
```suggestion
FROM quay.io/konveyor/builder AS builder
```

---

yes

I believe is better to change the minimal necessary, so we do not change the behavior, just add multi-arch support to the image